src/block/horizontal-scroller/frontend-horizontal-scroller.js

```diff
@@ -11,6 +11,14 @@ class StackableHorizontalScroller {
 		let initialClientX = 0
 
 		els.forEach( el => {
+			// get all <a> tags for inner columns
```

Comment:
While this comment says exactly what the line does, I prefer comments that explains what the chunk of code is for, or explain what it does for complex ones. For example:

```suggestion
			// Get all links, because we will need to disable them during drag
```